### PR TITLE
Support for Logitech G815 and G513

### PR DESCRIPTION
**Pull request by SamPoulton**
**List any issues that this PR fixes:** fixes # , etc...

None
**This pull request proposes the following changes:**

- Support for Logitech G815
- Support fro Logitech G513 as per #1778 

**Known issues/To do:**

- None

### CODE REVIEW

File: Project-Aurora/Aurora-LeagueLogiLEDWrapper/Aurora-LeagueLogiLEDWrapper.vcxproj [hunk 1]
```diff
@@ -22,32 +22,32 @@
...
     <RootNamespace>AuroraLeagueLogiLEDWrapper</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v141</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>
```
Comment (diogotr7) [inline Project-Aurora/Aurora-LeagueLogiLEDWrapper/Aurora-LeagueLogiLEDWrapper.vcxproj]: Please revert the toolset update.

Comment (SamPoulton) [reply Project-Aurora/Aurora-LeagueLogiLEDWrapper/Aurora-LeagueLogiLEDWrapper.vcxproj]: Yeah sorry about that, not quite sure what happened there but I've reverted it now

Comment (SamPoulton) [reply Project-Aurora/Aurora-LeagueLogiLEDWrapper/Aurora-LeagueLogiLEDWrapper.vcxproj]: I've really messed something up here, sorry

**Comment by SamPoulton** [comment]
You know what, I've completely messed this up, let's try again

**Comment by SamPoulton** [comment]
Okay it _should_ all be sorted now (please excuse my lack of git knowledge).

**Comment by SamPoulton** [comment]
Nope, time to start over.

**Comment by diogotr7** [comment]
if you need git help feel free to ask on aurora's discord server

**Pull request closed without merging**